Makefile

```diff
@@ -124,7 +124,7 @@ diff-check:
 
 .PHONY: tidy-deps
 tidy-deps:
-	go mod tidy
+	go mod tidy -compat=1.17
```

Comment:
We need to set this flag because of the change of go 1.17 module management.
https://go.dev/doc/go1.17#go-command